### DISCUSSION TITLE
tolgee with google translation [free API] [self hosted]

### DISCUSSION THREAD

**Discussion by zakidze** [tolgee/tolgee-platform#2264] [Q&A] [answered]
how to activate TOLGEE with self host and google translation free api [OPEN GOOGLE TRANSLATE NO API KEY]

**Comment by JanCizmar** [comment, marked as answer]
Hello,

you can find the info in the [self-hosting configuration docs](https://tolgee.io/platform/self_hosting/configuration#google-cloud-translation).
